sorc/checkout.sh

```diff
@@ -165,7 +165,7 @@ if [[ ${checkout_gsi} == "YES" ]]; then
 fi
 
 if [[ ${checkout_gdas} == "YES" ]]; then
-  checkout "gdas.cd" "https://github.com/NOAA-EMC/GDASApp.git" "aaf7caa"; errs=$((errs + $?))
+  checkout "gdas.cd" "https://github.com/NOAA-EMC/GDASApp.git" "feature/prepland"; errs=$((errs + $?))
```

Comment:
Note: to update this to whatever the stable nightly hash will be tomorrow AM.

Reply:
will update to the hash on Monday.

Reply:
GDASApp hash has been updated.